sjb/config/test_cases/test_branch_origin_extended_conformance_install_system_containers.yml

```diff
@@ -95,12 +104,18 @@ extensions:
       script: |-
         docker_image_tag="$( cat ./ORIGIN_TAG )"
         docker_repository="${OS_IMAGE_PREFIX:-"openshift/origin"}"
-        docker inspect "${docker_repository}:${docker_image_tag}"
+        [[ "$(sudo atomic containers list -n --no-trunc -f runtime=runc -f image=${docker_repository}:${docker_image_tag} --json | jq '.[0].running')" == "true" ]]
```

Comment:
@stevekuznetsov ah I understand what you're asking now. No, you can't use ``--json`` with JSONPath. It simply provides the data as JSON expecting other tools to manipulate the data if needed.